src/Appwrite/Utopia/Response/Filters/V12.php

```diff
@@ -81,7 +81,7 @@ public function parse(array $content, string $model): array
             case Response::MODEL_WEBHOOK_LIST:
             case Response::MODEL_KEY_LIST:
             case Response::MODEL_PLATFORM_LIST:
-            case Response::MODEL_DOMAIN_LIST:
+            // case Response::MODEL_DOMAIN_LIST:
```

Comment:
can we revert this file ?

Reply:
We dont have domain model anymore

Reply:
We can keep as an exception because it will be back in next version.